x-pack/plugins/infra/public/services/telemetry/telemetry_events.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import {
+  InfraTelemetryEventTypes,
+  InfraTelemetryEvent,
+  HostsViewQuerySubmittedSchema,
+  HostEntryClickedSchema,
+} from './types';
+
+const hostsViewQuerySubmittedSchema: HostsViewQuerySubmittedSchema = {
+  control_filters: {
```

Comment:
This is tricky. In the end both `filters` and `control_filter` are [`Filter`](https://github.com/elastic/kibana/blob/main/packages/kbn-es-query/src/filters/build_filters/types.ts#L72).

I think in the end, what both attributes will hold is an array of `Query`:
`[{'exists': { 'field': 'host.name' }},
   {'term': { 'host.os.name': 'Ubuntu' }}]` 

So probably `control_filters` should be typed as `text` too